python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -488,11 +488,12 @@ def func(x):
     # Just a single column -> result will be all NA
     data = cudf.Series([1, 2, cudf.NA])
     if constant is cudf.NA and op in {operator.pow, operator.ipow}:
-        # in pandas, 1**NA == 1. In cudf, 1**NA == 1.
-        with pytest.xfail():
+        # in pandas, 1**NA == 1. In cudf, 1**NA == NA. See
+        # https://github.com/rapidsai/cudf/issues/7478
```

Comment:
Pandas is more correct if one is to treat NA like nan.

Reply:
Many long discussions of this in the past. Short answer is that it's big problem which is also hard to solve because libcudf will never special case single values like this. So we end up having to introspect data in python which has historically been a non-starter. Won't hijack this PR but happy to discuss further.

Reply:
I note there _is_ precedent for post-hoc inspection (with integer division by zero). Arguably checking for NA is easier because we can just inspect `null_count` which doesn't require a scan of the column every time.

Reply:
Probably best to follow up on this issue separately from this PR.